api/ciprovider.go

```diff
@@ -0,0 +1,48 @@
+package api
+
+import (
+	"fmt"
+
+	"github.com/airware/vili/circleci"
+	"github.com/airware/vili/config"
+	"github.com/airware/vili/environments"
+	"github.com/airware/vili/kube"
+	"github.com/airware/vili/log"
+	"github.com/airware/vili/slack"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// InitializeCiClient initializes the CI client
+func InitializeCiClient(ciProvider string) error {
+	switch ciProvider {
+	case "circleci":
+		circleci.Init(&circleci.Config{
+			Token:   config.GetString(config.CircleciToken),
+			BaseURL: config.GetString(config.CircleciBaseurl),
+		})
+	case "":
```

Comment:
What if the environmental variable `CI_PROVIDER` is not set?  I think we should add a `default` and `nil` case to handle that....

Reply:
This is not a required parameter, so it won't panic. I updated the empty string case to a default case for now, until we integrate with other CIs as well.